css/src/new-dashboard.css

```diff
@@ -1,4 +1,92 @@
 body.toplevel_page_wpseo_dashboard {
+	.notice, .yoast-migrated-notice {
+		display:none;
+	}
+
+	.yoast-new-dashboard-notices:last-child{
+		margin-bottom: 2rem;
+	}
+
+	.yoast-dashboard-notice, .yoast-webinar-dashboard{
+		background: white;
+		border: 1px solid #E0B3CC;
+		box-shadow: 0px 4px 6px -1px #0000001A;
```

Comment:
Similarly, this might be easier to check for correctness if applying tailwind classes? 😅 

Border color `primary-200` ✅ 
But it seems the design has something else as shadow (`shadow-md`):
![image](https://github.com/user-attachments/assets/667841b2-166a-468b-bbf1-c250e3d3d364)